ships/0001-previous-proposals.md

```diff
@@ -0,0 +1,103 @@
+<!--
+Copyright The Shipwright Contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+---
+title: previous-proposals
+authors:
+  - "@adambkaplan"
+reviewers:
+  - "@ImJasonH"
+  - "@alicedoe"
+approvers:
+  - "@qu1queee"
+  - "@sbose78"
+creation-date: 2021-05-24
+last-updated: 2021-06-03
+status: implemented
+see-also:
+  - https://github.com/shipwright-io/build/tree/main/docs/proposals
+replaces: []
+superseded-by: []
+---
+
+# Previous Proposals
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [docs](/docs/)
+
+## Summary
+
+This is a summary of the merged proposals that pre-dated the SHIP process.
+Full details of these proposals can be found in [shipwright-io/build/docs/proposals](https://github.com/shipwright-io/build/tree/main/docs/proposals).
+
+## Motivation
+
+As of May 24, 2021, the community repository will serve as the official record of all new Shipwright enhancement proposals.
+Previously discussed and merged proposals will not be migrated here - they will remain in shipwright-io/build for posterity.
```

Comment:
What's the motivation to keep them on the controller repository? Wouldn't it be more practical to copy them over here (this repo) and keep them all together?

Reply:
(And after copying, removing them from the Build-Controller repo).

Reply:
Moving everything over presented its own challenges, particularly for proposals currently under discussion. Felt easier to keep the old in the controller repo, and have new proposals after 5/24/2021 live here.